data/json/hobbies.json

```diff
@@ -1,4 +1,59 @@
 [
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "driving_license",
+    "name": "Driving License",
+    "description": "You got your driving license and know how to drive a car.",
```

Comment:
I'd say "how to drive a car, change the oil and tires, and other basics", and add "mechanics 1".

It's also fine to start at driving 2 I think.